src/index.html

```diff
@@ -1,13 +1,58 @@
+<!--#region head-->
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+     content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin ="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
+  <!--#endregion-->
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image">
+        <img src="./images/apple-monoblock.png" alt="Apple monoblock">
+      </div>
+
+      <div class="card__content">
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina<br>
```

Comment:
Text could be different, do not use tags like`br`
```suggestion
          APPLE A1419 iMac 27" Retina
```